core/templates/base-components/base-content.directive.ts

```diff
@@ -29,9 +29,12 @@ angular.module('oppia').directive('baseContent', [
     return {
       restrict: 'E',
       scope: {},
-      bindToController: {},
+      bindToController: {
+        backButtonShown: '<'
+      },
       transclude: {
         breadcrumb: '?navbarBreadcrumb',
+        preLogoAction: '?navbarPreLogoAction',
```

Comment:
Why do you need to modify the base components? Can this be done is some other file, I assume this is only used in the new subtopic viewer and not in the other places.

Reply:
This is for adding a back button for the nav header. I thought it could be utilized on other pages in the future.

Reply:
Okay, makes sense.